backup.ps1

```diff
@@ -334,7 +336,15 @@ function Send-Email {
         # create a temporary error log to log errors; can't write to the same file that Send-MailMessage is reading
         $temp_error_log = $ErrorLog + "_temp"
 
-        Send-MailMessage @ResticEmailConfig -From $ResticEmailFrom -To $ResticEmailTo -Credential $credentials -Subject $subject -Body $body @attachments 3>&1 2>> $temp_error_log
+        #sender ([MimeKit.MailboxAddress] http://www.mimekit.net/docs/html/T_MimeKit_MailboxAddress.htm, required)
+        $From = [MimeKit.MailboxAddress]$ResticEmailFrom;
+        $RecipientList = [MimeKit.InternetAddressList]::new();
+        $RecipientList.Add([MimeKit.InternetAddress]$ResticEmailTo);
+        $Port = 465
```

Comment:
Port shouldn't be hardcoded IMHO. It would make more sense to add a variable in `secrets.ps1` e.g. `$PSEmailPort` and delete `$Port` here. You'd also delete `$ResticEmailConfig=@{UseSsl=$true; Port="587"}` in `secrets_template.ps1` since that's no longer used.